deploy/manifests/base/config.yaml

```diff
@@ -16,3 +16,4 @@ default-miner-config:
     retrieval-timeout: 1m0s
     max-concurrent-retrievals: 1
 miner-configs: {}
+event-recorder-endpoint-url: https://vtz1qw149h.execute-api.us-east-2.amazonaws.com
```

Comment:
Two comments.

Minor: This is environment-specific, i.e. dev us-east-2 and ideally we should move it out of the base kustomization overlay. If this value can be configured via environment variables, then it should be straight forward to set it only in `dev` overlay. Otherwise, I recommend creating a `replace`ment copy of the config in `dev` overlay. I suspect eventually we will need a dedicated config for that environment anyway. Please feel free to leave a TODO and approach in a separate PR.

I am not entirely familiar with the way the database is set up. In terms of traffic routing, it might be worth considering setting up a Private Link? That would keep the traffic off the internet as well as other benefits.